doc/source/admin/conda_faq.rst

```diff
@@ -0,0 +1,332 @@
+|                          |                          |                          |
+|                          |                          | IMPORTANT : Due to a     |
+|                          |                          | current limitation in    |
+|                          |                          | conda, the total length  |
+|                          |                          | of the                   |
+|                          |                          |                          |
+|                          |                          | conda\_prefix and the    |
+|                          |                          | job\_working\_directory  |
+|                          |                          | path should be less      |
+|                          |                          | than 50 characters!      |
++--------------------------+--------------------------+--------------------------+
+| ``conda_auto_install``   | False                    | Set to True to instruct  |
+|                          |                          | Galaxy to look for and   |
+|                          |                          | install Conda packages   |
+|                          |                          | for missing tool         |
+|                          |                          | dependencies before      |
+|                          |                          | running a job            |
++--------------------------+--------------------------+--------------------------+
+| ``conda_auto_init``      | False                    | Set to True to instruct  |
+|                          |                          | Galaxy to install Conda  |
+|                          |                          | automatically if it      |
+|                          |                          | cannot find a local copy |
++--------------------------+--------------------------+--------------------------+
+
+*Table 1: Important configuration options for Conda in Galaxy*
+
+
+2. How do Conda dependencies work? Where do things get installed?
+
+In contrast to the old dependency system, which was used exclusively by
+Galaxy, Conda is a pre-existing, independent project. With Conda it
+is possible for an admin to install packages without touching Galaxy at
+all – managing your dependencies independently from Galaxy. Galaxy can
+handle these dependencies for you, but admins are not required to use
+Galaxy for dependency management.
+
+There are a few new config options in the galaxy.ini file (see Table
+1 or ``galaxy.ini.sample`` for more information), but by default Galaxy
+will install Conda (the package manager) and the required packages in
+the ``<tool_dependency_dir>/_conda/`` directory. In this directory,
+Galaxy will create an ``envs`` folder with all of the environments managed
+by Galaxy. Every environment contains a ``lib``, ``bin``, ``share``, and ``include``
+subdirectories, depending on the tool, and is sufficient to get a Galaxy
```

Comment:
You could go either way with 'Every environment contains a <the list of directories> subdirectory' or 'Every environment contains <the list of subdirectories> subdirectories', but you can't use 'a' with 'subdirectories' here.